README.md

```diff
@@ -11,6 +11,15 @@ at the heart of the [Wikibase software](http://wikiba.se/).
 ### 3.0.0 (alpha)
 
 * Removed cloning from `MultiTerm.getTexts`.
+* Turned EntityId into a simple wrapper around an opaque serialization string
+  * Removed EntityId::getNumericId
+  * Removed EntityId::getEntityType
+  * Removed EntityId::getPrefixedId
+  * Introduced EntityId::getSerialization
```

Comment:
Not really:
* [Scope resolution operator in C++](https://en.wikipedia.org/wiki/Scope_resolution_operator#C.2B.2B)
* [Scope resolution operator in Ruby](https://en.wikipedia.org/wiki/Scope_resolution_operator#Ruby)
* [Quick access to a classes prototype in CoffeeScript](http://coffeescript.org/#classes)